src/Compiler/CompilerUtils.cpp

```diff
@@ -806,7 +819,7 @@ static int emitOutputFiles(std::string outputNameNoExt,
   case EmitLib: {
     std::string sharedLibNameWithExt;
     int rc = compileModuleToSharedLibrary(
-        module, outputNameNoExt, sharedLibNameWithExt);
+        module, outputNameNoExt, sharedLibNameWithExt, context);
```

Comment:
@imaihal just be noticed that `module` and `context` are potentially used later by `outputCode`. So please make sure `outputCode` still works since `module` has been released inside `compileModuleToSharedLibrary`.